src/test/rgw/bucket_notification/zone_bn.py

```diff
@@ -0,0 +1,196 @@
+import logging
+import ssl
+import urllib
+import hmac
+import hashlib
+import base64
+import xmltodict
+from http import client as http_client
+from urllib import parse as urlparse
+from time import gmtime, strftime
+from .multisite import Zone
```

Comment:
don't import any multisite code into this file

Reply:
Yes, for now I've just pasted them but I'll remove them. Thanks for pointing I'll note this down.

Reply:
note that you will have to fix the code accordingly, and change the interfaces of the methods, as they currently dependent on zones to get the RGW address

Reply:
Okay, will try to figure out this thing